tox.ini

```diff
@@ -96,6 +96,9 @@ deps =
     figure-!devdeps: matplotlib==3.3.2
     figure-!devdeps: mpl-animators==1.0.0
 
+    # To be fixed in a new pytest release:
+    py310: git+https://github.com/pytest-dev/pytest
```

Comment:
This doesn't seem like a great idea - why is this required?

Reply:
This is to fix the asdf fails on the CI.

Reply:
Can we skip the ASDF tests instead, conditional on python 3.10?

Reply:
Sure.